gui/build.py

```diff
@@ -118,19 +115,48 @@ def parse_args():
 
     bundle = subparsers.add_parser('bundle',
                                    help='bundle the application (freeze)')
+    if IS_WIN:
+        default_python_path = 'C:\\Python27'
+        bundle.add_argument('--nsis-path', dest='nsis_path',
+                            default='C:\\NSIS',
+                            help='your NSIS path on the'
+                            ' system(default: %(default)r)')
+    else:
+        default_python_path = '/usr/lib/python2.7'
+
     bundle.add_argument('--python-path', dest='python_path',
-                        default='C:\\Python27',
+                        default=default_python_path,
                         help='your python path on the'
                         ' system(default: %(default)r)')
-    bundle.add_argument('--nsis-path', dest='nsis_path',
-                        default='C:\\NSIS',
-                        help='your NSIS path on the'
-                        ' system(default: %(default)r)')
     bundle.set_defaults(func=do_bundle)
 
     return parser.parse_args()
 
 
+def call_cx_freeze(gui_path, moz_path, python_lib_path):
+    args = []
+    if IS_WIN:
+        args.append('--icon=wininst/app_icon.ico')
+        args.append('--base-name=Win32GUI')
+        args.append('--include-path=%s;%s;%s\site-packages;%s'
+                    % (gui_path, moz_path, python_lib_path, python_lib_path))
+        args.append('--target-name=mozregression-gui.exe')
+    else:
+        args.append('--include-path="%s:%s:%s:%s/dist-packages"'
+                    % (gui_path, moz_path, python_lib_path, python_lib_path))
+        args.append('--target-name=mozregression-gui')
+    args.append('--target-dir=dist')
+    args.append('mozregui/main.py')
+    if IS_WIN:
+        python_dir = os.path.join(
```

Comment:
I think it would be best to move this in the previous IS_WIN statement. The idea is that you can add all what you needs in args, then at the end just issue a:
call(*args)